CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
 - Fix: missing lastSuccess/lastFailure associated to initial notification on subscription creation some times when csub cache is in use (#2974)
 - Fix: several invalid memory accesses (based on a workaround, not a definitive solution, see issue #2994)
 - Add: release_date and doc fields are added to the GET /version output to align with FIWARE scheme (#2970)
-- Fix: broken JSON due to unscaped quotes (") in NGSIv2 error description field (#2955)
\ No newline at end of file
+- Fix: broken JSON due to unscaped quotes (") in NGSIv2 error description field (#2955)
+- Add: Partial CORS Preflight Requests support, -cors switch (#501)
```

Comment:
What means "partial" in this context? That this PR implements only a subset of resources (in particular on the `/v2` resource)? Of that the implemented stuff (i.e. adding some headers, etc.) is not enough to implement CORS preflight and something else will be needed in the future?

(Again probably a naïve-CORS question :) )

Reply:
I'd suggest to use:

```
- Add: CORS Preflight Requests support for /v2 and /v2/entities, -cors switch (#501)
```

and, for future PRs, new lines with:

```
- Add: CORS Preflight Requests support for <whichever URLs covered by the PR>, -cors switch (#501)
```

so, at the end, we can use all these lines to "checksum" if we have covered all the URLs in the NGSIv2 API, and, in that case, replace them all in a final PR with:

```
- Add: CORS Preflight support for all NGSIv2 resources, -cors switch (#501)
```

Reply:
Fixed in 543c64b